Foundation/NSTimeZone.swift

```diff
@@ -249,3 +292,7 @@ extension NSTimeZone {
     }
 
 }
+
+extension NSNotification.Name {
+    public static let NSSystemTimeZoneDidChange = NSNotification.Name(rawValue: "NSSystemTimeZoneDidChangeNotification") // NSUnimplemented
```

Comment:
As in make it read `public static let NSSystemTimeZoneDidChange = NSNotification.Name(rawValue: "")`, since that seems to be what the project does.